### ISSUE TITLE
`asyncData` does not work

### ISSUE THREAD

**Issue by TokugawaTakeshi** [IlyaSemenov/nuxt3-class-component#3]
Thank you for the this library.
Unfortunately, in my component that very similar to README.md example does not work:

`<template lang="pug">`
```pug
.KanjiAssignmentPage-SpecifcContent
  span {{ sample }}
  span {{ test }}
```
`</template>`

`<script lang="ts">`
```typescript
  import { Vue } from "vue-facing-decorator";

  @NuxtComponent({
    async asyncData() {
      console.log("ASYNC!!!");
      return { sample: "a" };
    }
  })
  export default class SamplePage extends Vue {

    protected sample!: string;
    protected test: string = "b";

    protected created(): void {
      console.log("CREATED");
    }

  }
```
`</script>`

The `ASYNC!!!` even does not displayed in the terminal.
Following the `README.md` I have set the `modules: [ "nuxt3-class-component" ]` to my `nuxt.config.ts`:

```ts
import { defineNuxtConfig } from "nuxt/config";
import Path from "path";


/* 【 参考 』 https://nuxt.com/docs/api/configuration/nuxt-config */
export default defineNuxtConfig({
  
  // ...

  modules: [ "nuxt3-class-component" ]

});

```

Have I missed something?

**Comment by CaptainSeven** [comment]
Had the same issue, I was able to at least get the example working by downgrading vue-facing-decorator to 2.1.20 which is what this lib uses. Seems that it doesn't work with the latest version of that.

**Comment by cypherix93** [comment]
Looks like all that is needed for this library to work with the latest `vue-facing-decorator` is a small change in this line:
https://github.com/IlyaSemenov/nuxt3-class-component/blob/master/src/runtime/imports/nuxt-component.ts#L26

before:
```typescript
return defineNuxtComponent(component)
```
after:
```typescript
return defineNuxtComponent(toNative(component))
```
The `toNative` is imported from `vue-facing-decorator`.

**Comment by IlyaSemenov** [comment]
@cypherix93 Thank you for investigating and coming up with a fix. It was published to npm in version 1.2.0. To be honest I didn't even test this, please let me know if everything works fine now.

What I still don't understand is how did this problem occur in the first place? The package used to explicitly tell that it pulled vue-facing-decorator@2:

https://github.com/IlyaSemenov/nuxt3-class-component/blob/5f37d40cdd708dde940733c9926582f4edb03a8e/package.json#L34

so how come you ended up with version 3?

**Comment by cypherix93** [comment]
@IlyaSemenov Thanks for merging the PR.

To clarify, I was already using `vue-facing-decorator` in my project. It was not until later (from @CaptainSeven's comment) that I realized that `nuxt3-class-component` was pointing to an older version of the library.

As for testing the change, I used your existing unit tests to verify that:
1. Without my change, all tests pass when using `vue-facing-decorator@2`
2. Updating `vue-facing-decorator` to version 3 makes all tests fail
3. With my change, on top of `vue-facing-decorator@3`, all tests pass again.

I also verified everything works in a barebones nuxt project. All in all, I think we're good. I'll keep an eye if anything goes awry.
